Code/application/Utilities/DataVariantFactoryImp.cpp

```diff
@@ -213,13 +213,13 @@ class DataVariantValue : public DataValueWrapper
 
    DataValueWrapper* copy() const
    {
-      DataValueWrapper* pData = new DataVariantValue(&(this->DISAMBIGUATE_TEMPLATE mValue));
+      DataValueWrapper* pData = new DataVariantValue(&(this->mValue)); // T is not a template, it's a typename or class
```

Comment:
This code was required when Solaris was a supported system because the STL was a bit older and had some issues requiring this. Removing it everywhere is fine now.